README.md

```diff
@@ -1,14 +1,14 @@
 # Markdown Tutorial 
 ![](http://i.imgur.com/IMTN5cy.png)  
 
-Hello, welcome to my tutorial for markdown.  
-In this tutorial you will learn the most basics things about Markdown.  
-Spanish version available [here](https://github.com/LewisVo/Markdown-Tutorial/blob/master/Translation:Spanish.md).
-Portuguese version available [here](https://github.com/LewisVo/Markdown-Tutorial/blob/master/README_pt-BR.md).
-French version available [here](https://github.com/luongvo209/Markdown-Tutorial/blob/master/README_fr.md).
+## Hello, welcome to my tutorial for markdown 👋.  
+In this tutorial you will learn the most basics things about Markdown. 👨‍🏫👩‍🏫
+- Spanish version available [here](https://github.com/LewisVo/Markdown-Tutorial/blob/master/Translation:Spanish.md). <img src="https://lh3.googleusercontent.com/proxy/sLHvxktCU13bf4b_UDFrpFeme4uQ5EF0_2e_BtnfQcvmOdlTWc4pShjke4kPYmfOCA2FEZFricSJfxKK9i3iQP2N" height="25px">
+- Portuguese version available [here](https://github.com/LewisVo/Markdown-Tutorial/blob/master/README_pt-BR.md). <img src="https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/160/apple/118/flag-for-portugal_1f1f5-1f1f9.png" height="25px">
+- French version available [here](https://github.com/luongvo209/Markdown-Tutorial/blob/master/README_fr.md).  <img src="https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/160/apple/114/flag-for-france_1f1eb-1f1f7.png" height="25px">
```

Comment:
@CodingSpecies, what do you think about using the emoji of the flags instead of these images? For example:

- Spain ( 🇪🇸 )
- Portugal ( 🇵🇹 )
- France ( 🇫🇷 )

Reply:
I agree with this suggestion. Moreover, as of the time I try to review this, the first image for Spanish is not loading at all.
Some general recommendations:
1. When possible, use emoji instead of raw image
2. If raw image is a must, paste it into Github to use Github as hosting server, not having to rely on third party server that we dont know will work forever

Reply:
Yes, I agree, I tried to search the emojis on my keyboard but I couldn't find it, I will do it now. thanks

Reply:
> @CodingSpecies, what do you think about using the emoji of the flags instead of these images? For example:
> 
> * Spain ( 🇪🇸 )
> * Portugal ( 🇵🇹 )
> * France ( 🇫🇷 )

Do I just put the 🇫🇷 and the rest like that in the place of the images?

Reply:
> > @CodingSpecies, what do you think about using the emoji of the flags instead of these images? For example:
> > 
> > * Spain ( es )
> > * Portugal ( portugal )
> > * France ( fr )
> 
> Do I just put the  and the rest like that in the place of the images?

It's just a suggestion, but I believe it's the best option because, in addition to being more standardized, it has the points that @luong-komorebi described.

Reply:
> Do I just put the 🇫🇷 and the rest like that in the place of the images?


Yes